### ISSUE TITLE
Node behavior in the layer tree (expand vs. load all containing layers)

### ISSUE THREAD

**Issue by RicardCots** [sitmun/sitmun-viewer-app#45]
**Is your feature request related to a problem? Please describe.**
Clicking on any folder containing layers in the layer tree will load all contained layers into the map. The administrator user should have the option to choose the behavior of a layer folder so that it can load all layers or just expand to show the list of contained layers.

**Describe the solution you'd like**
Add a property to tree nodes in the SITMUN admin / API SITMUN.

**Describe alternatives you've considered**

**Additional context**

**Comment by RicardCots** [comment]
Not exactly. The idea is to be able to decide whether the contents of a folder can be loaded onto the map as a single layer, or whether, on the contrary, only browsing the contents is allowed and each layer must be loaded individually. The WorkLayerManager component of the SITNA API distinguishes between the concept of a “single layer” and a “layer folder”. To clarify: if a layer group is defined in the service capabilities, the SITNA API allows it to be loaded into the loaded-layers component and treated as if it were a single layer.

In the following image, the loaded layers menu shows two different layers: one represents a layer group, while the other is a simple layer. Being able to load a set of layers as a single layer is very useful in all cases where multiple layers together form a map—for example, topographic maps or maps associated with planning and urban development instruments.

<img width="1574" height="745" alt="Image" src="https://github.com/user-attachments/assets/22cb41d5-140a-4899-b449-a203351ab003" />

The same in the original API SITNA

<img width="1557" height="775" alt="Image" src="https://github.com/user-attachments/assets/2a30d032-1e03-42b9-943a-b5dd1d80a494" />

**Comment by ibarzrm** [comment]
In version 1.2.1, it is still only possible to load layers individually.

**Comment by ibarzrm** [comment]
From DIBA, it is considered useful to add functionality to load a group of layers all at once. 

**Examples of possible solutions**: right-click, the folder icon or a specific button. Meanwhile, the folder title should open the folder's contents. 

 Contributions and confirmation of other opinions from the technical committee are awaited.

**Comment by fjlopez** [comment]
**May update**: Data model + API serialization are largely there; viewer implementation + admin UX are still missing to close the feature end-to-end.

**Comment by fjlopez** [comment]
Scope refactored after reviewing the behavior in SITMUN 2 and API SITNA:

- This issue now covers folder interaction semantics: browse-only folders versus folder activation, including “select all” checkboxes and default radio-button behavior.
- Loading a cartography folder as a single composite entry in SITNA’s `WorkLayerManager` has been moved to #166.
- Radio-button parity is largely implemented. The remaining work is to support the corresponding `loadData` configuration in the admin application and the viewer.
